src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/HandlePausePaymentsRequest.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+// ReSharper disable once RedundantUsingDirective
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.PausePayments;
+using SFA.DAS.EmployerIncentives.Commands.Exceptions;
+using SFA.DAS.EmployerIncentives.Domain.Exceptions;
+
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess
+{
+    public class HandlePausePaymentsRequest
```

Comment:
I think this http function should be in the SFA.DAS.EmployerIncentives.Functions project.  There is an existing HttpTriggerWithdrawalRequest function that this can use on the EI-677 branch.

Reply:
Thought about that and and I thought the Withdrawal requests are coming from the Employer. so therefore will need to be public facing. However the 'Pause/Resume Payments' are similar to the 'StartPayment Process' and  are coming from the compliance team and therefore do not want to be public facing.

Reply:
The ComplianceWithdrawl is in the functions project.  Best to check with Mike.

Reply:
This has been reworked so the functionality has been moved to the API layer and is called via the Functions app

Reply:
Can this function be removed from inner API now?

Reply:
yes it can, missed that